tests/suites/cloud/tests/multicontainer/index.js

```diff
@@ -112,7 +35,7 @@ module.exports = {
           );
 
         // Check that device env variable is present in each service
-        let services  = [`frontend`, `proxy`, `data`]
+        let services  = [`containerA`, `containerB`];
```

Comment:
Couldn't see it in the test, but just making sure where we are pushing this app.